Sources/PackageSigning/SignatureProvider.swift

```diff
@@ -310,6 +308,66 @@ struct CMSSignatureProvider: SignatureProviderProtocol {
             throw SigningError.unableToValidateSignature("\(error)")
         }
     }
+
+    func extractSigningEntity(
```

Comment:
@neonichu Implemented `extractSigningEntity` as part of this PR

Reply:
nice, thanks!